CHANGELOG.md

```diff
@@ -5,6 +5,37 @@
 ## [Unreleased]
 
 
+## [0.5.0]
+
+_2023-09-20_
+
+### Spark
```

Comment:
nit: for consistency, could we always use backticks when referring to class names?